pkg/v1/tkg/test/tkgctl/shared/package.go

```diff
@@ -693,3 +693,14 @@ func CheckTKGSAddons(ctx context.Context, tkgctlClient tkgctl.TKGClient, svClust
 
 	return nil
 }
+
+func ValidateClusterClassConfigFile(clusterclassConfigFilePath string) (string, string) {
```

Comment:
Question: We have same file name under ```pkg/v1/tkg/test/tkgctl/tkgs```. Which of these two is now duplicate and redundant and can be removed?

Reply:
potentially we could remove the other two instances and make sure they all use this one in a shared directory

Reply:
@maralavi yes I added here because don't want to call functions from other tests, let me clean up the redundancies in this PR and only defined in shared/package.go

Reply:
removed duplication function and let other test suite use package.go only